From 7ed5e15f2de747e383ba7eee66b646a7a66e6654 Mon Sep 17 00:00:00 2001
From: Sergey Kostanbaev <sergey.kostanbaev@gmail.com>
Date: Wed, 29 Dec 2021 12:30:12 -0800
Subject: [PATCH] Do not run qtdetail


qtdetail is a tool that generates qtdetail.out. Since this program is
cross-compiled, it's not possible to run it on the host.

Consequently, python-pyqt5.mk generates the qtdetail.out file before
calling configure.py.

Therefore, this patch makes sure that the pre-generated qtdetail.out
file is kept, and that qtdetail is not executed.

Signed-off-by: Gwenhael Goavec-Merou <gwenhael.goavec-merou@trabucayre.com>
Signed-off-by: Sergey Kostanbaev <sergey.kostanbaev@gmail.com>
Signed-off-by: Adam Duskett <aduskett@gmail.com>
Refresh for 5.15.6

diff --git a/configure.py b/configure.py
index daa0f94..32f5b21 100644
--- a/configure.py
+++ b/configure.py
@@ -2455,10 +2455,6 @@ def run_test_program(mname, test, verbose):
 
     out_file = 'cfgtest_' + mname + '.out'
 
-    # Create the output file, first making sure it doesn't exist.
-    remove_file(out_file)
-    run_command(test + ' ' + out_file, verbose)
-
     if not os.access(out_file, os.F_OK):
         error("%s failed to create %s. Make sure your Qt installation is correct." % (test, out_file))
 
-- 
2.33.1

